tests/test_podman_compose.py

```diff
@@ -43,7 +43,7 @@ def test_podman_compose_extends_w_file_subdir():
         "rmi",
         "--force",
         "localhost/subdir_test:me",
-        "docker.io/library/bash",
+        "docker.io/library/busybox",
```

Comment:
This test case verifies the use-case when a compose file's build context is outside of the current directory. In this test case the build context includes a Dockerfile which is based on the "busybox" image, not "bash".